README.md

```diff
@@ -0,0 +1,189 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#Hello-microverse)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-microverse] <a name="Hello-microverse"></a>
+
+**[Hello-microverse]** this was mini-projects that upkill to create a feature branch and test linters in order to have professional-looking code.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git </a></li>
+  </ul>
+</details
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your first "Hello-microverse" project
+
+To run this application localy, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+<br>
+-Node JS
+<br>
+-An IDE (Intergrated Development Environment) such vscode, atom etc ...
+<br>
+-web browser such as chrome, firefox etc ..
+
+### Setup
+
+Clone this repository to your desired directory:<br>
+Advantages of Linters:<br>
+
+1: improves readability-it will be easy to read and make adjustment if required easily.<br>
+2:removes silly errors before execution and code review.<br>
+3:Makes code look like written by a single person.<br>
+4:Spreads awareness and ownership over code quality<br>
+5:Having more secure and performant code.<br>
+
```

Comment:
- [ ] Kindly make sure you improve the content of the `Setup` sub section and add the `Usage` sub section with the appropriate information needed by users i order to clone and run your app locally. **This will make your application be more descriptive and professional**. You can proceed like this:
```
### Setup
To clone this project, run the following commands:
`cd your-prefered-folder-name`

`git clone https://github.com/migo1/Hello-Microverse.git`

### Usage
To run this app in your browser, run in your terminal:
`cd Hello-Microverse`

Then open the live server or open the `index.html` file with your browser 
.
.
.